src/chart/line/LineSeries.ts

```diff
@@ -120,6 +120,8 @@ export interface LineSeriesOption extends SeriesOption<LineStateOption<CallbackD
     data?: (LineDataValue | LineDataItemOption)[]
 
     triggerLineEvent?: boolean
+
+    closed?: boolean
```

Comment:
I think `loop` (inspired by `gl.LINE_LOOP` of WebGL) may be a better name than `closed` because it may not always be closed due to the existance of null data.